package-lock.json

```diff
@@ -8918,10 +8845,6 @@
         "url": "https://github.com/sponsors/ljharb"
       }
     },
-    "node_modules/fungible_token_interface_example": {
```

Comment:
@hugo-heer before you run `npm i` and update the package-lock, you need to have the packages built in your environment so that the package-lock matches what's in CI.

`STELLAR_SCAFFOLD_ENV=development stellar scaffold build --build-clients`
`npm i`